theme/color-themes/microbit-light.json

```diff
@@ -0,0 +1,118 @@
+{
```

Comment:
I plan to remove this theme before checking in, but it's been useful for testing